src/utils/VafLogger.nim

```diff
@@ -27,5 +29,5 @@ proc printResponse*(response: VafFuzzResult): void =
         urlDisplay = urlDisplay.replace(response.word, fmt"{resetcols}{khaki}{response.word}{resetcols}{orange}")
     if "200" == statusCode or "201" == statusCode:
         statusColor = lightgreen
-    log("result", &"{resetcols}{statusColor}Status: {statusCode}; Length: {response.responseLength}; Time: {response.responseTime}ms\t{response.word} {orange}{urlDecoded} {urlDisplay} {resetcols}")
+    log("result", &"{resetcols}{statusColor}Thread #{threadId}: Status: {statusCode}; Length: {response.responseLength}; Time: {response.responseTime}ms\t{response.word} {orange}{urlDecoded} {urlDisplay} {resetcols}")
```

Comment:
should be a `;`

Reply:
I was thinking `:` because thats like what the thread returned. Thread 1: status; length; time

Reply:
ow yeh that better